chat-widget/src/plugins/createChatTranscript.ts

```diff
@@ -80,13 +141,33 @@ class TranscriptHTMLBuilder {
         return htmlData;
     }
 
+    createScriptElement(src: string, integrity: string | undefined = undefined, crossOrigin: string | undefined = undefined, referrerPolicy: string | undefined = undefined) {
+        return `<script src="${src}" ${integrity ? `integrity="${integrity}"`: ""} ${crossOrigin ? `crossorigin="${crossOrigin}"`: ""} ${referrerPolicy ? `referrerpolicy="${referrerPolicy}"`: ""}><\/script>`;
+    }
+
     createExternalScriptElements() {
+        const webChatScript = this.externalScripts?.botframeworkWebChat?.src?
+            this.createScriptElement(this.externalScripts?.botframeworkWebChat?.src as string, this.externalScripts?.botframeworkWebChat?.integrity, this.externalScripts?.botframeworkWebChat?.crossOrigin, this.externalScripts?.botframeworkWebChat?.referrerPolicy):

```

Comment:
same, lets keep in mind single responsability principle, lets abstract this functionality , aiming for readability and maintenablity